websites/P/Puray Animes/presence.ts

```diff
@@ -7,23 +7,35 @@ presence.on("UpdateData", () => {
 		presenceData: PresenceData = {
 			largeImageKey: "logo",
 			startTimestamp: browsingTimestamp
-		};
+		},
+		video = document.querySelector("video");
 	if (document.querySelector("input[id^=headlessui]"))
 		presenceData.details = "Pesquisando Animes";
+	else if (pathname.startsWith("/login")) presenceData.details = "Logando";
+	else if (pathname.startsWith("/register"))
+		presenceData.details = "Cadastrando";
 	else if (pathname.startsWith("/home"))
-		presenceData.details = "Na página principal";
+		presenceData.details = "Página principal";
 	else if (pathname.startsWith("/profile/")) {
-		presenceData.details = "Visualizando perfil:";
+		presenceData.details = "Visualizando perfil";
 		presenceData.state = `${
 			document.querySelector("h3").childNodes[0]?.textContent
 		} - ${document.querySelector("button[class*=blue] span")?.textContent}`;
-		presenceData.buttons = [{ label: "Ver perfil", url: location.href }];
+		presenceData.buttons = [
+			{ label: "Ver perfil", url: location.origin + location.pathname }
```

Comment:
why?

Reply:
I was testing and forgot to revert.
I'll fix and commit.